lib/bundler/cli/gem.rb

```diff
@@ -231,6 +248,35 @@ def ask_and_set_test_framework
       test_framework
     end
 
+    def ask_and_set_ci
+      ci_template = options[:ci] || Bundler.settings["gem.ci"]
+
+      if ci_template.nil?
+        Bundler.ui.confirm "Do you want to add Continuous Integration to your gem? " \
+        "Adding a CI service to your project helps ensure your project is well tested " \
+        "before shipping your gem to users. Bundler recommends several different services for testing "\
```

Comment:
I think "before shipping your gem to users" could also be skipped?

Also, I would replace "Bundler recommends several different services for testing your code" with "Bundler's gem generator supports several different services for testing your code".